INTITA/Common/Extensions/UITextField+indent.swift

```diff
@@ -8,8 +8,11 @@
 import UIKit
 
 extension UITextField {
-    func indent(size:CGFloat) {
-        self.leftView = UIView(frame: CGRect(x: self.frame.minX, y: self.frame.minY, width: size, height: self.frame.height))
+    func indent(size: CGFloat) {
+        self.leftView = UIView(frame: CGRect(x: self.frame.minX,
+                                             y: self.frame.minY,
+                                             width: size,
+                                             height: self.frame.height))
         self.leftViewMode = .always
```

Comment:
Is it possible to have this piece of code with less `self` statements?

Reply:
It has done.